marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/extensions/runFlow.sjs

```diff
@@ -19,27 +19,46 @@ const datahub = new DataHub();
 
 //todo flush this out
 function get(context, params) {
-  let flowName = params["flowName"];
+  post(context, params, null);
+}
 
-  let flow = null;
+function post(context, params, input) {
+  let flowName = params.flowName;
 
-  if(!fn.exists(flowName)) {
-      fn.error(null,"RESTAPI-SRVEXERR",  Sequence.from([400, "Bad Request", "Invalid request - must specify a flowName"]));
+  if (!fn.exists(flowName)) {
+    fn.error(null,"RESTAPI-SRVEXERR",  Sequence.from([400, "Bad Request", "Invalid request - must specify a flowName"]));
   }
-  else{
-    flow = datahub.flow.getFlow(flowName);
-    let uris = params['identifier'];
-
+  else {
+    let options = params.options ? JSON.parse(params.options) : {};
+    let jobId = params.jobId || datahub.hubUtils.uuid();
+    let flow = datahub.flow.getFlow(flowName);
+    let query = null;
+    let uris = null;
+    if (params.uri) {
+      uris = datahub.hubUtils.normalizeToArray(params.uri);
+      query = cts.documentUriQuery(uris);
+    } else {
```

Comment:
Is this meant to be "cts.documentQueryUris" ?

Reply:
Is that supposed to be cts.documentQuery?